Makefile

```diff
@@ -45,27 +47,19 @@ server-shell: compose-build
 	@docker-compose run $(IMAGE_NAME) /bin/bash
 
 integration-shell: compose-build
-	# note- does not bring up external connectors such as redshift or snowflake
-	@echo "Bringing up main image and images for integration testing"
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml up -d
-	@echo "Waiting 15s for integration containers to be ready..."
-	@sleep 15
-	@echo "Running additional setup for mssql integration tests"
-	@docker exec -it fidesops python tests/integration_tests/mssql_setup.py
-	@docker-compose -f docker-compose.yml -f docker-compose.integration-test.yml run $(IMAGE_NAME) /bin/bash
+	@virtualenv -p python3 fidesops_test_dispatch; \
+		source fidesops_test_dispatch/bin/activate; \
+		python run_infrastructure.py --open_shell --datastores $(datastores)
```

Comment:
Interesting that you need the (kinda ugly) `virtualenv` wrapper here. Assuming you *need* a venv, how about this instead: `./fidesops_test_dispatch/bin/python3 run_infrastructure.py ...`

This just uses the venv's python to run the command and is something I've seen in other projects.

You could also have a *classic* make target called `integration-venv` that just creates ./fidesops_test_dispatch.

So to summarize:
```suggestion
integration-shell: compose-build integration-venv
    ./fidesops_test_dispatch/bin/python3 run_infrastructure.py --open_shell --datastores $(datastores)
    
# Create the venv as needed    
integration-venv:
  virtualenv -p python3 fidesops_test_dispatch
```

Reply:
Oh actually even better, you can leverage makes own intelligence to only run the target when needed...
```
integration-shell: compose-build fidesops_test_dispatch/bin/python3
    ./fidesops_test_dispatch/bin/python3 run_infrastructure.py --open_shell --datastores $(datastores)
    
# Create the venv as needed    
fidesops_test_dispatch/bin/python3:
  virtualenv -p python3 fidesops_test_dispatch
```

Reply:
If you read the logs in Github actions the virtualenv command actually fails there. This is required for local running where users have `python` set to invoke Python 2 — open to suggestions to make this better in a follow up